addons/osfstorage/models.py

```diff
@@ -164,7 +164,7 @@ def is_preprint_primary(self):
     def delete(self, user=None, parent=None, **kwargs):
         self._path = self.path
         self._materialized_path = self.materialized_path
-        return super(OsfStorageFileNode, self).delete(user=user, parent=parent) if self._check_delete_allowed() else None
+        return super().delete(user=user) if self._check_delete_allowed() else None
```

Comment:
This is a drive by clean up [here's the TODO](https://github.com/CenterForOpenScience/osf.io/pull/9593/files#diff-b427522b7934e79065d8524342c6f63f34c40f3a1ff2712a36c05466b14639adL413)